aio/src/styles/2-modules/search-results/_search-results-theme.scss

```
@@ -29,7 +29,7 @@
       }
 
       .no-results {
-        color: mat.get-color-from-palette($foreground, text);
+        color: #fff;
```

Please change this to:
```suggestion
        color: constants.$white;
```